autoPyTorch/pipeline/components/training/trainer/base_trainer.py

```diff
@@ -185,22 +203,34 @@ def prepare(
         metrics_during_training: bool,
         scheduler: _LRScheduler,
         task_type: int,
-        labels: Union[np.ndarray, torch.Tensor, pd.DataFrame]
+        labels: Union[np.ndarray, torch.Tensor, pd.DataFrame],
+        step_unit: Union[str, StepIntervalUnit] = StepIntervalUnit.batch
     ) -> None:
 
+        step_unit_types = [str, StepIntervalUnit]
+        if not any(isinstance(step_unit, step_unit_type) for step_unit_type in step_unit_types):
```

Comment:
```suggestion
        if isinstance(step_unit, step_unit_types):
```
you can check if an object is in an iterable of types directly.

Reply:
Ok, I will change to the following:
```
if not isinstance(step_unit, step_unit_types):
```